artic/README.md

```diff
@@ -4,11 +4,11 @@ title: Artic
 
 ## Live Resources
 
-| usegalaxy.eu | usegalaxy.org.au |
-|:--------:|:------------:|
-| <FlatShield label="data" message="view" href="https://usegalaxy.eu/library/list#folders/F2a32545de855d335" alt="Raw data" /> | |
-| <FlatShield label="workflow" message="run" href="https://usegalaxy.eu/u/wolfgang-maier/w/covid19-variation-analysis-on-artic-pe-data" alt="Galaxy workflow" /> | <FlatShield label="workflow" message="run" href="https://usegalaxy.org.au/u/simongladman/w/covid-19-variation-analysis-on-artic-pe-data" alt="Galaxy workflow" />  |
-| <FlatShield label="history" message="view" href="https://usegalaxy.eu/u/wolfgang-maier/h/artic-example-analysis" alt="Galaxy history" /> | |
+| usegalaxy.org | usegalaxy.eu | usegalaxy.org.au |
+|:-------------:|:-----------:|:----------------:|
+| | <FlatShield label="data" message="view" href="https://usegalaxy.eu/library/list#folders/F2a32545de855d335" alt="Raw data" /> | |
+| <FlatSheild label="workflow" message="run" href="https://usegalaxy.org/u/aun1/w/covid-19-variation-analysis-on-artic-pe-data-1" alt="Galaxy workflow" />| <FlatShield label="workflow" message="run" href="https://usegalaxy.eu/u/wolfgang-maier/w/covid19-variation-analysis-on-artic-pe-data" alt="Galaxy workflow" /> | <FlatShield label="workflow" message="run" href="https://usegalaxy.org.au/u/simongladman/w/covid-19-variation-analysis-on-artic-pe-data" alt="Galaxy workflow" />  |
```

Comment:
Thanks for taking the initiative in updating this info @bwlang :rocket:
There has been so much development happening on all the covid-19 WFs that nobody had the time to keep the project page up to date :(

There's a typo in `FlatShield`, but besides that:

The current .eu WF link would be:
https://usegalaxy.eu/u/wolfgang-maier/w/covid19-variation-analysis-on-artic-pe

For Australia, I'm not 100% sure, but probably:
https://usegalaxy.org.au/u/galaxy-australia/w/covid-19-variation-analysis-on-artic-pe-data-imported-from-uploaded-file
Would that be correct, @Slugger70?

On .eu and .org, @mvdbeek is planning to reupload the dockstore/iwc versions of all WFs under an actual iwc account so that in the future there will be an easier way to locate the "official" WFs.

Reply:
@wm75 Yes, that's it. I updated it this morning (my time). I need to rename them I guess.

Reply:
@mvdbeek Feel free to create an IWC account on .org.au also. We'd be happy about that. But we can continue to publish them under galaxy-australia if you all prefer.

Reply:
💯 already did that, just need to set up the automation

Reply:
any chance to update the links here? 
I forgot that its not yet here and pointed people to usegalaxy.org in a seminar today - whups!
let me know how I can help.